jenkins-slaves/jenkins-slave-cpp/README.md

```diff
@@ -0,0 +1,20 @@
+# jenkins-slave-cpp
+Provides a docker image of the C++ runtime for use as a Jenkins slave.
+
+## Build local
+`docker build -t jenkins-slave-cpp .`
+
+## Run local
+For local running and experimentation run `docker run -i -t jenkins-slave-cpp /bin/bash` and have a play once inside the container.
+
+## Build in OpenShift
+```bash
+oc process -f ../templates/jenkins-slave-generic-template.yml \
```

Comment:
I think this needs to be `oc process -f ../.openshift/templates/jenkins-slave-generic-template.yml`

Reply:
@pabrahamsson thx for the review and yes you are correct. It is fixed now.